packages/api/refresh.go

```diff
@@ -83,14 +83,16 @@ func getRefreshTokenClaims(w http.ResponseWriter, data *apiData, logger *log.Ent
 		logger.WithFields(log.Fields{"type": consts.JWTError}).Error("getting jwt claims")
 		return nil, errorAPI(w, `E_TOKEN`, http.StatusBadRequest)
 	}
-	token, err := jwt.ParseWithClaims(data.params[`token`].(string), &JWTClaims{},
```

Comment:
that one was better

Reply:
call of jwt.ParseWithClaims more clear when he is inline